packages/trader/src/AppV2/Components/EmptyPositions/empty-positions.tsx

```diff
@@ -13,7 +13,7 @@ const EmptyPositions = ({ isClosedTab, noMatchesFound }: TEmptyPositionsProps) =
     const [showLoader, setShowLoader] = React.useState(true);
 
     React.useEffect(() => {
-        const timeout = setTimeout(() => setShowLoader(false), 500);
+        const timeout = setTimeout(() => setShowLoader(false), 700);
```

Comment:
Increased, cause after moving contract type  filtration to BE we need to give more time for such cases as rare contract for all time